src/Components/login/Login.js

```diff
@@ -20,8 +20,16 @@ const Login = () => {
   const [showPassword, setShowPassword] = useState(false);
   const [isLoggedIn, setIsLoggedIn] = useState(false);
   const [goCreateAccount, setGoCreateAccount] = useState(false);
+
   const { classes } = styles();
 
+  useEffect(() => {
+    const informationUser = JSON.parse(localStorage.getItem("userInformation"));
```

Comment:
lets create a function called, `getInfoFromLocalStorage` move that function inside utils.js file and export it from there, then import in here and call the `getInfoFromLocalStorage` on line 27. Something like this:
`const informationUser =  getInfoFromLocalStorage("userInformation");`